lib/diagnostic.js

```diff
@@ -6,13 +6,16 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (splitWords) {
+  let words = "Today Is The Day After Wednesday.";
+  let results = words.split(" ");
 };
```

Comment:
So close! You are on the right track, but you forgot to `return` the array of words.